onnxruntime/test/python/onnx_backend_test_series.py

```diff
@@ -163,7 +163,6 @@ def create_backend_test(testname=None):
                                  '^test_compress_*',
                                  '^test_constant_pad_cpu.*',
                                  '^test_gemm_default_scalar_bias_cpu.*',
-                                 '^test_gather_negative_indices_cpu.*',
```

Comment:
We need to remove all the tests corresponding to the newly added kernels from this list and make sure it passes. For example, Unsqueeze, Squeeze, Slice, etc.

Reply:
Also any remnants within main.cc under the onnx_test_runner project will have to be removed from exclusion and makes sure they pass in the CI check.